Alternative Visualizations/Ethnocentrism - Alternative Visualization.nlogo

```diff
@@ -363,7 +363,7 @@ mutation-rate
 mutation-rate
 0.0
 1.0
-0.0050
```

Comment:
I've seen diffs like this a few times. They don't seem to cause any problems in practice. I suppose they're caused by the JVM update...